frontend/src/views/SqlEditor/EditorPanel/QueryEditor.vue

```diff
@@ -47,6 +48,17 @@ const selectedInstanceEngine = computed(() => {
   ) as string;
 });
 
+const selectedLanguage = computed(() => {
+  const engine = selectedInstanceEngine.value;
+  if (engine === "MySQL") {
```

Comment:
Should also work with TiDB

Reply:
> Should also work with TiDB

The Monaco Editor `basic-languages` now does not support TiDB, So besides `mysql`/`pgsql`, other languages now use the default `sql` as the Monaco Editor language.

We may define a new language to support TiDB later, but not in this PR.

Reply:
TiDB declares it's MySQL compatible, that's why we can use it (though there maybe surprises, ok to do it separately)